routes/alumni.js

```diff
@@ -14,4 +14,16 @@ router.get('/', function(req, res, next) {
   })
 });
 
+router.post('/', function(req, res, next) {
+  var newAlumni = new alumni(req.body.newAlumni);
+  newAlumni.save(function(err, results){
+      if(err) {
+          console.log(req.body);
```

Comment:
for this path we can do `console.error("got an error for ", req.body, "error message: ", err)`

Reply:
added console.error for path